### ISSUE TITLE
Model weights and hyperparameters

### ISSUE THREAD

**Issue by DarthXerox** [Spiideo/soccersegcal#1]
Hello, 
I'm asking if you are planning to upload final weights of the learned model. It would be very helpful to me and definitely to other people trying out this model later on. 
Also, could you please elaborate on hyperparameters of your best model? Which learning rate did you choose, how many epochs did the model require to reach the final result or what batch size was optimal for you.

Thank you!

**Comment by hakanardo** [comment]
Sure thing! I'm on vacation right now but I'll look into it when I get
back...

**Comment by DarthXerox** [comment]
Alright, that would be great! Thanks for a swift response

**Comment by hakanardo** [comment]
Hyperparameters and weights are now presented at the bottom of the readme.
